pkg/cli/repositories.go

```diff
@@ -0,0 +1,589 @@
+			if err != nil {
+				return err
+			}
+		}
+		return nil
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+
+		pathCheck, err := checkKeysFolder()
+		if err != nil {
+			SetError(err, 1)
+			return
+		}
+
+		if len(args) > 0 {
+			// Set default to args
+			key := args[0]
+			f, err := os.Open(filepath.Join(pathCheck, key))
+			if err != nil {
+				SetError(fmt.Errorf("%s does not exist as a key stored", key), 3)
+				return
+			}
+			defer f.Close()
+			data, err := ioutil.ReadAll(f)
+			if err != nil {
+				SetError(fmt.Errorf("unable to read from key file"), 4)
+				return
+			}
+
+			defaultF, err := os.OpenFile(filepath.Join(pathCheck, "default"), os.O_RDWR|os.O_CREATE, 0644)
+			if err != nil {
+				SetError(fmt.Errorf("unable to open default key: %s", err.Error()), 5)
+				return
+			}
+
+			defer defaultF.Close()
+			err = ioutil.WriteFile(filepath.Join(pathCheck, "default"), data, os.ModePerm)
+			if err != nil {
+				SetError(err, 6)
+				return
+			}
+
+			// Finished doing what we wanted
+			return
+		}
+
+		// Open default
+		f, err := os.Open(filepath.Join(pathCheck, "default"))
+		if err != nil {
+			SetError(errors.New("default key has not been set"), 6)
+			return
+		}
+		defer f.Close()
+
+		h := md5.New()
+		if _, err := io.Copy(h, f); err != nil {
+			SetError(err, 7)
+			return
+		}
+
+		fis, err := ioutil.ReadDir(pathCheck)
+		if err != nil {
+			SetError(err, 8)
+			return
+		}
+
+		for _, fi := range fis {
+
+			if fi.Name() != "default" {
+				f2, err := os.Open(filepath.Join(pathCheck, fi.Name()))
+				if err != nil {
+					SetError(err, 9)
+					return
+				}
+				h2 := md5.New()
+				if _, err := io.Copy(h2, f2); err != nil {
+					SetError(err, 6)
+					return
+				}
+				if bytes.Equal(h.Sum(nil), h2.Sum(nil)) {
+					fmt.Println(fi.Name())
+				}
+				f2.Close()
+			}
+
+		}
+	},
+}
+
+var createKeyCmd = &cobra.Command{
+	Use:   "create NAME TOKEN",
+	Short: "Creates a file containing the access token to be referenced using name",
+	Args:  cobra.MaximumNArgs(2),
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 2 {
+			return errors.New("must provide a NAME and TOKEN")
+		}
+		usr, err := os.UserHomeDir()
+		if err != nil {
+			return err
+		}
+		pathCheck := filepath.Join(usr, ".vorteil", "repository-keys")
+		_, err = os.Stat(pathCheck)
+		if err != nil {
+			err = os.MkdirAll(pathCheck, os.ModePerm)
+			if err != nil {
+				return err
+			}
+		}
+		return nil
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		name := args[0]
+		if name == "default" {
+			SetError(errors.New("default is a reserved word and can't be a name"), 1)
+			return
+		}
+		key := args[1]
+
+		pathCheck, err := checkKeysFolder()
+		if err != nil {
+			SetError(err, 2)
+			return
+		}
+
+		// Check if file exists
+		// if stat returns no error return error saying you need to provide the force flag
+		fi, err := os.Stat(filepath.Join(pathCheck, name))
+		if err == nil {
```

Comment:
Keep code complexity down, also keep in-mind this will hide any errors on the os.Stat command, such `Insufficient Perm` error. Should only be a problem on certain edge cases though.
```suggestion
		if err == nil && !flagForce{
```